lib/boombox.ex

```diff
@@ -188,6 +185,22 @@ defmodule Boombox do
     end
   end
 
+  @doc """
+  Runs boombox with given input and plays audio and video streams on your computer.
+
+  Idiomatic to `Boombox.run(input: ..., output: :player)`.
+
+  ## Example
+
+  ```
+  Boombox.play(input: "rtmp://localhost:5432")
+  ```
+  """
+  @spec play(Enumerable.t() | nil, input: input() | stream_input()) :: :ok
+  def play(stream \\ nil, opts) do
+    run(stream, Keyword.put(opts, :output, :player))
+  end
```

Comment:
Why not like this:

````suggestion
  Boombox.play("rtmp://localhost:5432")
  ```
  """
  @spec play(Enumerable.t() | nil, input() | stream_input()) :: :ok
  def play(stream \\ nil, input) do
    run(stream, input: input, output: :player)
  end
````